index.js

```diff
@@ -407,6 +407,72 @@ module.exports = {
         if (err) return cb(clarify(err, 'Error finding or creating additions feed when starting ssb-tribes2'))
         return cb()
       })
+
+      // look for new epochs that we're added to
+      ssb.metafeeds.findOrCreate((err, myRoot) => {
+        // prettier-ignore
+        if (err) return cb(clarify(err, 'Error getting own root in start()'))
+
+        pull(
+          ssb.db.query(
+            // TODO: does this output new stuff if we accept an invite to an old epoch and then find additions to newer epochs?
+            where(and(isDecrypted('box2'), type('group/add-member'))),
+            live({ old: true }),
+            toPullStream()
+          ),
+          // groups/epochs we're added to
+          pull.filter((msg) => {
+            return msg.value?.content?.recps?.includes(myRoot.id)
```

Comment:
```suggestion
            return isGroupAdd(msg) && msg.value.content.recps.includes(myRoot.id)
```

Reply:
checked against the spec in a filter before this so removed the ?s